cluster/gce/config-default.sh

```diff
@@ -114,9 +114,33 @@ KUBE_UP_AUTOMATIC_CLEANUP=${KUBE_UP_AUTOMATIC_CLEANUP:-false}
 
 # OpenContrail networking plugin specific settings
 NETWORK_PROVIDER="${NETWORK_PROVIDER:-none}" # opencontrail, flannel
-OPENCONTRAIL_TAG="${OPENCONTRAIL_TAG:-R2.20}"
-OPENCONTRAIL_KUBERNETES_TAG="${OPENCONTRAIL_KUBERNETES_TAG:-master}"
-OPENCONTRAIL_PUBLIC_SUBNET="${OPENCONTRAIL_PUBLIC_SUBNET:-10.1.0.0/16}"
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+
+if [[ $NETWORK_PROVIDER == "opencontrail" ]]; then
+    img=$(gcloud compute images list | grep debian-7-backports | awk '{print $1}')
```

Comment:
Thanks for taking time to review. 

The params are not redefined. Its just defaulted to a value. 

About 5 rebases back, debian 7 wheezy backports was not working as the image itself was changed and did not have support for cgroup memory in the default grub. I see that updated now. 

Removed the default values for these now.